helpers.js

```diff
@@ -9,11 +10,24 @@ module.exports = {
           .setFooter('This is a community-made open source bot - if it goes down, fix it')
           .setThumbnail('https://cdn.discordapp.com/icons/333376110329069578/b5cf0e90b74a3e1b828c15a262b7a5c8.png?size=128')
           .setTimestamp();
-
-        message.channel.send(embed);
+        return embed;
       } catch (error) {
         logger.error('Failed to send embed message', error);
         message.channel.send('There was a problem, sorry!');
       }
     },
```

Comment:
Something is off with the indentation here - it may be that lines 2 and 3 have the wrong indentation, or that this line does, but this section definitely has indentation issues.

Reply:
I'm not going to point the blame for this one ;)